### PR TITLE
feat: add `Get Chat Settings` endpoint

### PR DESCRIPTION
**Pull request by Nerixyz**
Adds the [`Get Chat Settings`](https://dev.twitch.tv/docs/api/reference#get-chat-settings) endpoint to helix.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,10 @@
 
 * Deprecate `Get User` `view_count`
 
+### Added
+
+* Added `Get Chat Settings` endpoint
```
Comment (Nerixyz) [inline CHANGELOG.md:22]: I'll rebase/merge when #267 is merged.

**Comment by Emilgardis** [comment]
bors d+

**Review by Emilgardis — APPROVED** [review]
nice!

**Comment by Nerixyz** [comment]
Oops, I'll fix the docs.

**Pull request merged**